make.js

```diff
@@ -41,11 +42,19 @@ target.build = function() {
 
 target.units = function() {
     target.build();
+    var nodeVer = process.versions.node;
+    if(semver.lt(nodeVer,'8.0.0')){
+        pushd('_build');
+        run('npm install ../_build');
+        popd();
+    }
```

Comment:
Could you provide some context on why we should use `pushd('_build')` for Node 8?

Reply:
Versions below Node 8 have different installation behaviour, and work incorrectly with current test script, thus this change.